core/whistle_couch-1.0.0/src/couch_replicator.erl

```diff
@@ -50,16 +66,15 @@ repl_db(Source, Target, DB) ->
         %% bigcouch, it can get Source and Target talking...
 
         lager:debug("Replication beginning"),
-        case couch_util:db_replicate(Source, ReplData) of % if Source and Target can talk directly
+        case couch_util:db_replicate(AdminSource, ReplData) of % if Source and Target can talk directly
             %%case couch_mgr:db_replicate(ReplData) of
-            {ok, _} -> ok;
-            {error, {500, _}} ->
-                {error, failed}
+            {'ok', _} -> 'ok';
+            {'error', {500, _}} ->
+                {'error', 'failed'}
         end
     catch
         _E:_R ->
-            lager:debug("Exception occurred, probably a gateway timeout"),
-            lager:debug("~p", [_E]),
-            lager:debug("~p", [_R]),
-            {error, failed}
+            lager:error("Exception occurred: ~p:~p", [_E, _R]),
```

Comment:
@siplabs 
Really minor request, but since I want to discuss the patch to coucbeam, if you are going to change this line could you use a lowercase log line?   When this module was written we didn't have a standard logging policy but by convention we normalized on lowercase.  This has the result of making uppercase logging stick out like a sore thumb... might as well fix it while you are here :)